kratos/tests/test_flux_based_redistance.py

```diff
@@ -0,0 +1,200 @@
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+from KratosMultiphysics import python_linear_solver_factory as linear_solver_factory
+import KratosMultiphysics.vtk_output_process as vtk_output_process
+import math
+import os
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
```

Comment:
I remember there was some util that does exactly this, but I didn't manage to find it. @philbucher do you remember where it was placed?